config/newrelic.yml

```diff
@@ -52,16 +52,18 @@ common: &default_settings
 development:
   <<: *default_settings
   app_name: 'Open Food Network (Development)'
-  monitor_mode: false
+  agent_enabled: false
 
 test:
   <<: *default_settings
   # It doesn't make sense to report to New Relic from automated test runs.
   monitor_mode: false
+  agent_enabled: false
 
 staging:
   <<: *default_settings
   app_name: 'Open Food Network (Staging)'
+  agent_enabled: false
```

Comment:
How are we going to test changes before they go to production? It would be great to have this on staging as well. And testing locally?

Reply:
you can override the configuration by setting the env variable NEW_RELIC_AGENT_ENABLED to true